src/sync.rs

```diff
@@ -56,11 +58,24 @@ pub fn determine_posts(
                 break 'tweets;
             }
         }
-        // The tweet is not on Mastodon yet, let's post it.
-        updates.toots.push(NewStatus {
-            text: tweet_unshorten_decode(tweet),
-            attachments: tweet_get_attachments(tweet),
-        });
+
+        // The tweet is not on Mastodon yet, next step
+
+        // Fetch the tweet text into a String object
+        let decoded_tweet = tweet_unshorten_decode(tweet);
+        // Fetch the sync hashtag from the Option object for comfort
+        let sync_hashtag_twitter_value = options.sync_hashtag_twitter.as_deref().unwrap_or("");
```

Comment:
we should change this to `match options.sync_hashtag_twitter` and then have 2 arms for the None and for the Some case.